src/css/announcement-email.css

```diff
@@ -0,0 +1,107 @@
+.announcement-email {
+  background-color: var(--redpanda-900);
+  color: white;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  font-size: 21px;
+  overflow: hidden;
+  padding: 15px;
+  transition: opacity 0.4s ease-in-out;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: 999;
+  opacity: 0;
```

Comment:
How come we chose to use opacity 0 instead of display:none ?

Reply:
For the animation, it uses the opacity.